draft-pda-protocol.md

```diff
@@ -504,12 +493,6 @@ request:
 * the key config specifies a KEM, KDF, or AEAD algorithm the client doesn't
   recognize.
 
-[OPEN ISSUE: @chris-wood: Can't the leader determine if helpers are "online"?
-This seems to reveal information that's specific to clients. Imagine, for
-example, that clients are prohibited from talking to helpers but not the leader.
-Is it OK that leaders learn that about a client? I'm not sure, so I'd be
-inclined to remove this unless we have a concrete use case.]
-
```

Comment:
IIUC the concern here is that leaders or helpers can learn which clients are uploading data, and I believe deployments can address that by using OHTTP.